### PR TITLE
Add Clear Linux distro support

### PR DESCRIPTION
**Pull request by bryteise**

### CODE REVIEW

**Comment by rnesius** [comment]
bryteise and myself were added to an amendment to the CLA between Microsoft and Intel weeks ago.

File: config/clearlinux/waagent.conf [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+Logs.Verbose=n
+
+# Root device timeout in seconds.
+OS.RootDeviceScsiTimeout=300
+
+# If "None", the system default version is used.
+OS.OpensslPath=None
+
+# Enable or disable self-update, default is enabled
+AutoUpdate.Enabled=n
```
Comment (brendandixon) [inline config/clearlinux/waagent.conf]: @bryteise AutoUpdate.Enabled must be "y" by default. Customer, of course, may change it, but all files we ship with the agent should set it to enabled.

Comment (bryteise) [inline config/clearlinux/waagent.conf]: Fixed in d4a3226.

**Comment by brendandixon** [comment]
@bryteise Have you submitted a CLA?

**Comment by brendandixon** [comment]
@bryteise Scratch that. I just saw the comment above.

**Pull request merged**